study-html/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Progate</title>
+    <link rel="stylesheet" href="stylesheet.css">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="header-left">
+          <img class="logo" src="https://prog-8.com/images/html/advanced/main_logo.png">
+        </div>
+        <div class="header-right">
+          <a href="#" class="login">ログイン</a>
+        </div>
+      </div>
+    </header>
+    <div class="top-wrapper">
+      <div class="container">
+        <h1>LEARN TO CODE.</h1>
+        <h1>LEARN TO BE CREATIVE.</h1>
+        <p>Progateはオンラインプログラミング学習サービスです。</p>
+        <p>初心者にもやさしいスライドとレッスンで、ウェブサービスを作りながらプログラミングを学んでいきましょう。</p>
+        <div class="btn-wrapper">
+          <a href="#" class="btn signup">新規登録はこちら</a>
+          <p>or sign up with</p>
+          <a href="#" class="btn facebook"><span class="fa fa-facebook"></span>Facebookで登録</a>
+          <a href="#" class="btn twitter"><span class="fa fa-twitter"></span>Twitterで登録</a>
+        </div>
+      </div>
+    </div>
+    <div class="lesson-wrapper">
+      <div class="container">
```

Comment:
`container` といった名前は他とかぶる可能性が高いので、ユニークになるようにプレフィックスを付けましょう
```suggestion
      <div class="lesson-container">
```